Cannot install standalone cccorelib wrapper - Failed to checkout tag

Hi tmontaigu,
I get a fatal error when trying to install the standalone cccorelib wrapper from the git master branch:

```
  fatal: reference is not a tree: be52fc2f9981a80cd457cd914f44f17f6ebf04f1
  CMake Error at cccorelib-subbuild/cccorelib-populate-prefix/tmp/cccorelib-populate-gitclone.cmake:40 (message):
    Failed to checkout tag: 'be52fc2f9981a80cd457cd914f44f17f6ebf04f1'

```

I managed to install pycc from the master on my Linux ubuntu-20.04.2.0 machine.

--- a/wrapper/cccorelib/CMakeLists.txt
+++ b/wrapper/cccorelib/CMakeLists.txt
@@ -53,7 +53,7 @@ if(CCCORELIB_PYTHON_IS_MASTER_PROJECT)
     CCCoreLib
     GIT_REPOSITORY https://github.com/CloudCompare/CCCoreLib
     GIT_PROGRESS ON
-    GIT_SHALLOW ON
+    GIT_SHALLOW OFF
     GIT_TAG be52fc2f9981a80cd457cd914f44f17f6ebf04f1
   )
 

